tests/integration/telemetry/policy/testdata/enable_envoy_local_ratelimit.yaml

```diff
@@ -9,10 +9,6 @@ spec:
       app: clt
   configPatches:
     - applyTo: HTTP_FILTER
-      listener:
-        filterChain:
-          filter:
-            name: "envoy.http_connection_manager"
```

Comment:
This was removed because it will start failing test?
Or because we will start rejecting the config?

Reply:
`listener` was never a field in EnvoyFilter here. It could have been `foobar` and the same result. The problem was Envoyfilter previously didn't reject unknown fields (while all other types did) - now it does

Reply:
So this will break users CI/CD pipelines, even if it does not break behaviour.